test/services/s3.spec.js

```diff
@@ -2541,6 +2541,36 @@ describe('AWS.S3', function() {
     });
   });
 
+  if (typeof Promise === 'function') {
+    return describe('getSignedUrlPromise', function() {
+      var catchFunction, err;
+      err = null;
+      catchFunction = function(e) {
+        err = e;
+      };
+      beforeEach(function() {
+        AWS.util.addPromises(AWS.S3, Promise);
+        err = null;
+      });
+      it('resolves when getSignedUrl is successful', function() {
```

Comment:
This test is actually returning a false positive. The `catchFunction` is being called, but since there are no `expect` calls within it that can fail, mocha assumes the test passes.